b827ebfffe87de3a

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "b827ebfffe87de3a",
+    "servers": [
+      {
+        "server_address": "ttn-router-us-west",
```

Comment:
This is not the proper server address for the us-west router, check the valid routers here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses